scripts/download.py

```diff
@@ -16,3 +17,18 @@ def download_original(wd: str) -> None:
             print("Done")
         else:
             print("Original implementation found. Skipping download.")
+
+
+def download_from_hub(repo_id: Optional[str] = None, model_size: str = "7B") -> None:
```

Comment:
I find `model_size` in the args slightly misleading here, because it's not clear it will be used just to name the folder and not do other checks. As in, you may use a repo containing the 13B weights, and specify 7B in model_size and it will all be fine.

Personally I'd just expose `local_dir` here.

Reply:
`convert_hf_checkpoint.py` takes a model size as it will be used to write the path `checkpoints/lit-llama/{model_size}`

I thought this was a nice addition as it means that you only need to change the `--model_size` argument across all scripts

If we remove this then you'd need:
`convert_hf_checkpoint.py --checkpoint_dir whatever-maybe-32B --model_size 32B`
which is one extra argument.

But I can remove it if you prefer. Just wanted to be consistent across scripts

Reply:
Actually, since we want to set `local_dir` to avoid having users reach deep into the HF cache, I need a name for the folder to avoid conflicts with multiple downloads.

Would renaming `model_size` to `name` be better?